src/js/contentscript.js

```diff
@@ -17,19 +19,20 @@ const getSelector = hostname => {
           'tr.js-navigation-item > td.icon, .files-list > a.list-item',
         host: 'github',
       };
-    case 'gitlab.com':
-      return {
-        filenameSelector: 'tr.tree-item > td.tree-item-file-name',
-        iconSelector: 'tr.tree-item > td.tree-item-file-name > i',
-        host: 'gitlab',
-      };
     case 'bitbucket.org':
       return {
         filenameSelector: 'tr.iterable-item > td.filename > div > a',
         iconSelector: 'tr.iterable-item > td.filename > div > a > span',
         host: 'bitbucket',
       };
     default:
+      if (hostname.match(gitlabHostRegexp)) {
```

Comment:
I think you're after something more like `getSiteType` rather than `getHostname` since the hostname of GitHub Enterprise and GitLab can't be determined ahead of time and hard-coded (necessarily).